src/app/learn/get-started/prereqs/initial-setup/index.html

```diff
@@ -12,27 +13,27 @@
   <stache-page-anchor>
     Node.js
   </stache-page-anchor>
-
-  <p>
-    Install the latest LTS release of <a href="https://nodejs.org/en/download/releases/">Node.js</a>. We require at least version 6. To check your version, run <stache-code>node –v</stache-code> from the command line.
-  </p>
   <p>
-    For Mac OS X and Linux, we recommend that you use <a href="https://github.com/creationix/nvm">Node Version Manager (nvm)</a> to wrap your NodeJS installation so that it installs in your user directory and avoids permissions-related issues.
+    SKY UX requires an LTS Dubnium version of Node.js, and we recommend <a href="https://nodejs.org/en/download/releases/">the latest release</a>. To check your version, run <stache-code>node –v</stache-code> from the command line.
```

Comment:
I know we talked about this IRL John, but seeing it makes me think... what do you think about something like the following?

```
SKY UX highly recommends the <a href="https://nodejs.org/en/download/releases/">latest LTS Dubnium version</a> of Node.js.
```

Reply:
Sounds good. Updating now.